actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -77,6 +79,9 @@
     @Wired
     Container container;
 
+    @Inject
```

Comment:
Instead of the metrics manager approach wouldn't it be more flexible and modular to use the extensions system? Would also be easier to setup for those not using Orbit container.